tests/unit/Libraries/ResourceCache/ViewCacheTest.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace Quantum\Tests\Libraries\ResourceCache;
+
+use Quantum\Libraries\ResourceCache\ViewCache;
+use Quantum\Tests\AppTestCase;
+use Quantum\Loader\Setup;
+
+class ViewCacheTest extends AppTestCase
+{
+	private $viewCache;
+
+	private $route = '/current-route';
+
+	private $sessionId = 'session-id-123';
+
+	private $content = 'test html content';
+
+	private $ttl = 100;
+
+	public function setUp():void
+	{
+		parent::setUp();
+		if (!config()->has('view_cache')){
+			config()->import(new Setup('config', 'view_cache'));
+		}
+		config()->set('resource_cache', true);
+		$this->viewCache = new ViewCache();
+	}
+
+	public function testStoreViewCache()
+	{
+		$this->viewCache->set($this->route, $this->content, $this->sessionId);
+
+		$viewCache = $this->viewCache->get($this->route, $this->sessionId, $this->ttl);
+
+		$this->assertIsString($viewCache);
+		$this->assertEquals($this->content, $viewCache);
+	}
+
+	public function testGetViewCache()
+	{
+		$this->viewCache->set($this->route, $this->content, $this->sessionId);
+		$viewCache = $this->viewCache->get($this->route, $this->sessionId, $this->ttl);
+
+		$this->assertIsString($viewCache);
+		$this->assertEquals($this->content, $viewCache);
+	}
+
```

Comment:
The two tests above are the same, if there are no code-wise differences in these tests, maybe remove one of them?